template.yaml

```diff
@@ -806,3 +812,145 @@ Resources:
               - kms:ReEncrypt*
             Resource:
               - !GetAtt DatabaseKmsKey.Arn
+
+#
+# Method Management
+#
+
+  ######################################
+  # Method Management API Stub API
+  ######################################
+  MethodManagementv1StubsRestApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: method-management-api-stub
+      Description: HTTP API to simulate the Method Management API in non-prod environments
+      StageName: !Ref Environment
+      CacheClusterEnabled: true
+      CacheClusterSize: "0.5"
+      MethodSettings:
+        - HttpMethod: "*"
+          CachingEnabled: true
+          ResourcePath: /*
+      TracingEnabled: true
+      AccessLogSetting:
+        DestinationArn: !GetAtt MethodManagementv1ApiGatewayApiLogGroup.Arn
+
+  MethodManagementv1ApiGatewayApiLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/apigateway/${MethodManagementv1StubsRestApi}"
+      RetentionInDays: 1
+      KmsKeyId: !GetAtt LoggingKmsKey.Arn
+
+  ######################################
+  # MethodManagement API Stub Domain
+  ######################################
+  MethodManagementv1CustomDomain:
+    Type: AWS::ApiGatewayV2::DomainName
+    Properties:
+      DomainName: !Sub method-management-v1-stub.home.${Environment}.account.gov.uk
```

Comment:
We're waiting for clarification from Auth about if this API will share the domain with the existing Account Management API, or if it'll have a separate one like you've set up here. 

I don't think finding that out is a blocker for merging this PR as we can always come back and update either this or our frontend.

Reply:
Discussed with @alex9smith and @aenciso and decided to continue using the new sub-domain.  If auth decide to deploy on the same domain as they currently use then it will be a small config change.